target/linux/ramips/dts/mt7621_planex_vr500.dts

```diff
@@ -80,13 +80,39 @@
 	};
 };
 
-&ethernet {
+&gmac0 {
 	mtd-mac-address = <&factory 0xe000>;
 };
 
+&switch0 {
+	ports {
+		port@0 {
+			status = "okay";
+			label = "lan1";
+		};
+		port@1 {
+			status = "okay";
+			label = "lan2";
+		};
+		port@2 {
+			status = "okay";
+			label = "lan3";
+		};
+		port@3 {
+			status = "okay";
+			label = "lan4";
+		};
+		port@4 {
+			status = "okay";
+			label = "wan";
+			mtd-mac-address = <&factory 0xe006>;
+		};
+	};
+};
+
 &state_default {
 	gpio {
-		groups = "i2c", "uart2", "uart3", "rgmii2", "sdhci";
+		groups = "sdhci";
```

Comment:
All these changes only due to switch changes?
To be honest, I'd personally separate the state-default changes from the rest of this patch, so this only migrates the switch config from 01_leds/02_network to DTS.